src/units.h

```diff
@@ -65,44 +68,65 @@ namespace Faunus {
      *
      * Property           | Unit
      * :----------------- | :--------------------------
-     * Energy             | Thermal energy (kT)
-     * Temperature        | Kelvin (K)
-     * Length             | Angstrom (A)
-     * Charge             | Electron unit charge (e)
-     * Dipole moment      | Electron angstrom (eA)
-     * Concentration      | Particles / angstrom^3
-     * Pressure           | Particles / angstrom^3
-     * Angle              | Radians
+     * energy             | thermal energy unit (kT)
+     * temperature        | kelvin (K)
```

Comment:
Kelvin, Ångstrom, Dalton should start with capital letters. For mass, I prefer "grams per mole". Also, from Wikipedia on Dalton:

_2019 redefinition of the SI base units[edit]: 
The definition of the dalton was not affected by the 2019 redefinition of SI base units,[28][29][1] that is, 1 Da in the SI is still 1/12 of the mass of a carbon-12 atom, a quantity that must be determined experimentally in terms of SI units. However, the definition of a mole was changed to be the amount of substance consisting of exactly 6.02214076×1023 entities and the definition of the kilogram was changed as well. As a consequence, the molar mass constant is no longer exactly 1 g/mol, meaning that the number of grams in the mass of one mole of any substance is no longer exactly equal to the number of daltons in its average molecular mass.[30]_

Reply:
The easier part first. The unit names like kelvin, ångström, and dalton should start with a lowercase letter, unlike their abbreviations K, Å, and Da that start with an uppercase letter. A proof: https://goldbook.iupac.org/terms/view/K03374. You know, I am a big fan of typography hence I pay disproportionally huge attention to these tiny details. ;)

It is shame that the dalton was not redefined accordingly. Then g/mol appears to be the natural unit of choice instead. However, I struggle mentally a bit to use intensive units (per mole) to extensive quantities like mass or energy (we have already done it in the `kJmol` unit) as it leads to count in moles of moles. Nevertheless, it is only internal Faunus issue without any outside impact hence it can be resolved easily later. For now, I will use gram per mole as the internal unit of mass.